### DISCUSSION TITLE
Airtable-Query Result Empty.

### DISCUSSION THREAD

**Discussion by tareq-media** [Budibase/budibase#3681] [Help]
Hi,

I think the connection setup is done correctly with "API Key" & "base ID", still no result is returned when I run the query.

Please help.

Thanks

**Comment by mjashanks** [comment]
Hi @tareq-media 

I just successfully connected to Airtable using API Key & Base ID. Note that the table name and view name are case sensitive.

**Reply by tareq-media** [reply]
Hi @mjashanks,

Thanks for your reply. I am sure that all the info used are correct, still it is not returning any result.!!

**Comment by christianroy** [comment]
Same issue here. Hosted Budibase, the Airtable integration config seems ok, but I get the "Query results empty..." error whenever I try to add and run a query.

**Reply by christianroy** [reply]
arhgg... resolved it myself. The Airtable integration config was not ok. I had enterer the base _name_ in the "Base" field, fixed it by using the base id instead. My bad (although the interface could be clearer and explain what is expected in that field...)
